Dockerfile

```diff
@@ -14,6 +17,8 @@ RUN yum -y install gcc-c++ && \
     yum clean all && \
     rm --force ns.rpm
 
+RUN echo complete -C '/usr/bin/aws_completer' aws >> /root/.bashrc
```

Comment:
Gotcha! Yeah, I think to ultimately get this branch merged, we'll want to strip it down to its bare essentials for operation to ensure that folks using this as a reference understand what's needed.